health/egov-persister/plan-service-persister.yml

```diff
@@ -153,14 +159,16 @@ serviceMaps:
             - jsonPath: $.PlanConfiguration.auditDetails.lastModifiedBy
             - jsonPath: $.PlanConfiguration.auditDetails.lastModifiedTime
 
-        - query: INSERT INTO health.plan_configuration_operations(id, input, operator, assumption_value, output, active, show_on_estimation_dashboard, plan_configuration_id, created_by, created_time, last_modified_by, last_modified_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) ON CONFLICT (id) DO UPDATE SET input = ?, operator = ?, assumption_value = ?, output = ?, active = ?, show_on_estimation_dashboard = ?, last_modified_by = ?, last_modified_time = ?;
+        - query: INSERT INTO health.plan_configuration_operations(id, input, operator, assumption_value, output, source, category, active, show_on_estimation_dashboard, plan_configuration_id, created_by, created_time, last_modified_by, last_modified_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) ON CONFLICT (id) DO UPDATE SET input = ?, operator = ?, assumption_value = ?, output = ?, active = ?, show_on_estimation_dashboard = ?, last_modified_by = ?, last_modified_time = ?;
```

Comment:
why is there three question marks

Reply:
1 question mark got missed in previous pr, hence added it now.